src/main/java/com/google/sps/authentication/UserAuthenticationStatus.java

```diff
@@ -0,0 +1,74 @@
+// Copyright 2019 Google LLC
```

Comment:
2020?

I don't think this copyright is needed in all the files

Reply:
removed.